modules/ocf_irc/templates/services.conf.erb

```diff
@@ -0,0 +1,243 @@
+}
+
+networkinfo {
+    networkname = "OCF"
+
+    nicklen = 32
+    userlen = 10
+    hostlen = 64
+    chanlen = 32
+
+    modelistsize = 100
+    vhost_chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789.-"
+    allow_undotted_vhosts = false
+    disallow_start_or_end = ".-"
+}
+
+options {
+    casemap = "rfc1459"
+    seed = <%= @passwords['seed'] %>
+
+    strictpasswords = yes
+    badpasslimit = 5
+    badpasstimeout = 1h
+
+    updatetimeout = 5m
+    expiretimeout = 30m
+    readtimeout = 5s
+    warningtimeout = 4h
+    timeoutcheck = 3s
+    retrywait = 60s
+
+    hideprivilegedcommands = no
+    hideregisteredcommands = no
+
+    regexengine = "regex/posix"
+}
+
+
+# Load services config files
+include {
+    type = "file"
+    name = "chanserv.conf"
+}
+
+include {
+    type = "file"
+    name = "global.conf"
+}
+
+include {
+    type = "file"
+    name = "nickserv.conf"
+}
+
+include {
+    type = "file"
+    name = "operserv.conf"
+}
+
+
+# Log information to file and to the #services channel
+log {
+    target = "services.log #services"
+    bot = "Global"
+    logage = 60
+
+    admin = "*"
+    override = "chanserv/* nickserv/* memoserv/set ~botserv/set botserv/*"
+    commands = "*"
+    servers = "*"
+    users = "connect disconnect nick"
+    other = "*"
+    rawio = yes
+    debug = yes
+}
+
+log {
+    target = "globops"
+    admin = "global/* operserv/mode operserv/kick operserv/akill operserv/s*line operserv/noop operserv/jupe operserv/oline operserv/set operserv/svsnick operserv/svsjoin operserv/svspart nickserv/getpass */drop"
+    servers = "squit"
+    users = "oper"
+    other = "expire/* bados akill/*"
+}
+
+
+# Define different types of operators
+opertype {
+    name = "Services Operator"
+    commands = "chanserv/list chanserv/suspend chanserv/topic memoserv/staff nickserv/list nickserv/resetpass nickserv/suspend operserv/mode operserv/chankill operserv/szline operserv/akill operserv/session operserv/modlist operserv/sqline operserv/oper operserv/kick operserv/ignore operserv/snline"
+    privs = "chanserv/auspex chanserv/no-register-limit memoserv/* nickserv/auspex nickserv/confirm"
+}
+
+opertype {
+    name = "Services Administrator"
+    inherits = "Services Operator"
+    commands = "chanserv/access/list chanserv/drop chanserv/getkey chanserv/saset/noexpire memoserv/sendall nickserv/saset/* nickserv/getemail operserv/news operserv/jupe operserv/svsnick operserv/stats operserv/oline operserv/noop operserv/forbid global/*"
+}
+
+opertype {
+    name = "Services Root"
+    commands = "*"
+    privs = "*"
+    modes = "+Nq"
+}
+
+
+# Services opers
+oper {
+    name = "waf"
+    type = "Services Root"
+}
+
+oper {
+    name = "nattofriends"
+    type = "Services Root"
+}
+
+oper {
+    name = "ckuehl"
+    type = "Services Root"
+}
+
+oper {
+    name = "jvperrin"
+    type = "Services Root"
+}
+
+oper {
+    name = "mattmcal"
+    type = "Services Root"
+}
```

Comment:
Looks like I was bored! I only had one section to replace since we aren't using LDAP for IRCd opers, so it was much simpler and I could just use a regular array instead of an array of hashes.